lib/interface_quda.cpp

```diff
@@ -8132,7 +8101,15 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
   loopInfo.Nmoms = GK_Nmoms;
   int Nmoms = GK_Nmoms;
   char filename_out[512];
+  // Stochastic method should be fixed to 0
   int smethod = loopInfo.smethod;
+  if(smethod != 0) {
+    printfQuda("\nYour stochastic method is deprecated for MG\n");
+    printfQuda("enabled stochastic solutions. Switching to\n");
+    printfQuda("stochastic method 0.\n");
+    smethod = 0;
+  }
+
```

Comment:
I believe that the smethod option is redundant in the way that the code is constructed right now. I suggest that we just remove it from everywhere.